config/autocert_test.go

```diff
@@ -64,15 +64,15 @@ func TestAutocertOptions_Validate(t *testing.T) {
 		cleanup func()
 	}
 	tests := map[string]func(t *testing.T) test{
-		"ok/custom-ca": func(t *testing.T) test {
+		"ok/custom-ca": func(_ *testing.T) test {
```

Comment:
Note to self: it looks wrong to have a `func(*testing.T)` where `T` is unused, but it looks like this is set up so that some specific test cases can perform extra initialization steps (and the test should fail if these initialization steps fail). I think this could be refactored to make the intent clearer; filed https://github.com/pomerium/pomerium/issues/5100 to track separately.